core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -1373,10 +1373,29 @@ public UpdateCenter getUpdateCenter() {
         return updateCenter;
     }
 
+    /**
+     * If usage statistics has been disabled
+     *
+     * @since TODO
+     */
+    public Boolean isNoUsageStatistics() {
```

Comment:
Just to make it explicit why the method is different

```suggestion
    @CheckForNull
    public Boolean isNoUsageStatistics() {
```